server/.env

```diff
@@ -7,5 +7,7 @@ FHIR_API_URL=https://demo.arkhn.com/api
 USER_INFO_URL=http://localhost:4444/userinfo
 OAUTH2_CLIENT_ID=pyrog-server
 OAUTH2_TOKEN_URL=http://127.0.0.1:4444/oauth2/token
+SUPERUSER_EMAIL=admin@arkhn.com
```

Comment:
it may be confusing to add these lines to the .env (one may think that it is required to run pyrog-server). I would remove these two lines and document how to use the seed:superuser command in the README, what do you think ?